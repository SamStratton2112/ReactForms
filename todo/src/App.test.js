import { render } from '@testing-library/react';
import App from './App';

it('renders', () => {
  render(<App />);
});

it('should match snapshot', () => {
  const {asFragment}=render(<App />);
  expect(asFragment()).toMatchSnapshot();
});

