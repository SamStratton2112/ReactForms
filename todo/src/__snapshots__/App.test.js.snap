// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <form>
        <label
          for="todo"
        >
          To Do
        </label>
        <input
          id="todo"
          name="todo"
          placeholder="What to "
          type="text"
          value=""
        />
        <button>
          Add To Do
        </button>
      </form>
      <ul />
    </div>
  </div>
</DocumentFragment>
`;
